src/Datagrid/Datagrid.php

```diff
@@ -31,12 +31,12 @@ final class Datagrid implements DatagridInterface
     /**
      * The filter instances.
      *
-     * @var array
+     * @var array<string, FilterInterface>
```

Comment:
wdyt about this syntax vs FilterInterface[]? I like this one better (more explicit on the array keys) but since it is the first Pr I open about narrowing the api, better ask you.

Reply:
FilterInterface[] is meaning array<mixed, FilterInterface>.

The more explicit we can be, the better it is IMHO. So I vote array<string, FilterInterface>

Reply:
Ok, then I leave it as is.